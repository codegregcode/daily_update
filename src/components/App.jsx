import Landing from './Landing';
import '../styles/App.css';

function App() {
  return (
    <>
      <Landing />
    </>
  );
}

export default App;
