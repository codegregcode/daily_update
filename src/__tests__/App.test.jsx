import { render } from '@testing-library/react';
import App from '../components/App';

describe('App component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
