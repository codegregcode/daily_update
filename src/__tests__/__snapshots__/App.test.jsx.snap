// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > matches the snapshot 1`] = `
<DocumentFragment>
  <h1>
    daily_update
  </h1>
  <button>
    Login
  </button>
  <button>
    Register
  </button>
  <button>
    Forgotten Password?
  </button>
</DocumentFragment>
`;
